app/src/main/java/com/bleyl/recurrence/database/DatabaseHelper.java

```diff
@@ -119,6 +123,13 @@ public void onUpgrade(SQLiteDatabase database, int oldVersion, int newVersion) {
                     + COL_FRIDAY + " TEXT, "
                     + COL_SATURDAY + " TEXT) ");
         }
+
+        if (oldVersion < 4) {
+            database.execSQL("ALTER TABLE " + NOTIFICATION_TABLE + " ADD " + COL_NAG_TIMER + " INTEGER");
+            database.execSQL("ALTER TABLE " + NOTIFICATION_TABLE + " ADD " + COL_ACTIVE_STATE + " BOOLEAN");
+            database.execSQL("UPDATE " + NOTIFICATION_TABLE + " SET " + COL_NAG_TIMER + " = 0;");
+            database.execSQL("UPDATE " + NOTIFICATION_TABLE + " SET " + COL_ACTIVE_STATE + " = FALSE;");
```

Comment:
Hm, did a clearing data again and it works.

Probably Android of my device (Android 6.0) restored the database from backup.

Nonetheless this seems to be a problem to me if it crashes when a previous database will lead to crash. The ALTER TABLE-command will not be executed correctly with a previous database.

Nonetheless this commit is wrong in my eyes as SQLite does not have a BOOLEAN datatype. Booleans have to be stored as Integer with 0 = false and 1 = true

http://www.sqlite.org/datatype3.html